api/controllers/auth.controller.js

```diff
@@ -1,7 +1,35 @@
-module.exports.postRegister = (req, res) => {
-  res.json(req.body);
+const User = require('../models/user.model');
+const bcrypt = require('bcryptjs');
+
+module.exports.postRegister = async (req, res) => {
+	const { email, password } = req.body;
+
+	//Check if the user exists
+	const existingUser = User.findOne({ email });
+	if (existingUser) {
+		res.status(401).send('User has already exists');
```

Comment:
chỗ ni cu Trung set biến 401 = UNAUTH_STATUS nè,
mà anh nghĩ ví dụ như có mail rồi thì mình nên trả về 400 á, 400 là BAD_REQUEST_STATUS
ví dụ như ri nè
```suggestion
		return res.status(BAD_REQUEST_STATUS).send('User has already exists');
```
Nếu được thì cu Trung tạo cái folder constants nữa, rồi thêm một file là httpStatus.constant.js là max oke hehe

Reply:
là 1 folder constants, trong file httpstt là chứa kiểu BAD_REQUEST_STATUS=400 ri hả

Reply:
nhu ri ne
constants/httpStatus.constant.js

`
export const BAD_REQUEST_STATUS = 400;
export const UNAUTHORIZED_STATUS = 401;
`

roi qua kia vi du can xai la minh se 
`
import { UNAUTHORIZED_STATUS, BAD_REQUEST_STATUS } from '../constants/httpStatus.constant';
`